.github/workflows/destroy-preview-environment.yml

```
@@ -0,0 +1,84 @@
+                owner: 'ballerine-io',
+                repo: 'cloud-infra-config',
+                event_type: 'destroy-preview',
+                client_payload: {
+                  'ref': '${{ needs.deploy-dev-pr-environment.outputs.env_name }}'
+                }
+              });
+              console.log('Successfully triggered deploy-preview event');
+            } catch (error) {
+              console.error('Failed to trigger deploy-preview event:', error);
+              throw error;
+            }
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)